data/pages-content/faq.md

```diff
@@ -41,7 +54,7 @@ We are always keen to hear new ideas on how to enrich the programme and other ac
 
 ## **Q. When do you accept talk proposals?**
 
-A. The Call for Proposals (CFP) will be launched in early March. We are thrilled to receive proposals on any topic that could be of interest to our community.
+A. The Call for Proposals (CFP) will run for two weeks between 6-19 March 2023. We are thrilled to receive proposals on any topic that could be of interest to our community.
```

Comment:
Might be a good idea to add a line saying that they can reach out to programme@europython.eu, should they have any questions/ suggestions.